public/index.html

```diff
@@ -60,6 +60,10 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
 
     <!-- stylesheet -->
+    <!-- <link
+      rel="stylesheet"
+      href="https://unpkg.com/mockman-js@latest/dist/style.css"
+    /> -->
```

Comment:
Bhaiya, You can remove commented code.

Reply:
It is optional code. Sometime I may need it